robottelo/config/validators.py

```diff
@@ -29,9 +32,8 @@
         Validator('server.ssh_password', default=None),
     ],
     content_host=[
-        Validator('content_host.rhel_versions', must_exist=True),
         Validator('content_host.default_rhel_version', must_exist=True),
-        Validator('content_host.deploy_workflow', must_exist=True),
+        Validator('content_host.deploy_workflow.default', must_exist=True),
```

Comment:
If its must exist the we should remove the dummy test from `workflow-name` from the YAML otherwise we will have failed test cases with wrong workflow at least locally as we have a default value for it in CI yaml for content_host.

Reply:
removed field from the template. ultimately doesn't matter, user needs to make updates to the templates after copying them. If they don't the test fails exactly as expected when the tower workflow call fails. (unless they happened to name their workflow the same as the example).